src/app/Fake.Runtime/CompileRunner.fs

```diff
@@ -92,6 +92,17 @@ let tryRunCached (c:CoreCacheInfo) (context:FakeContext) : RunResult =
     | None -> RunResult.SuccessRun c.Warnings
     | Some e -> RunResult.RuntimeError e
 
+/// options that must be added to compilations to circumvent compilation errors from the DependencyManager preview feature
+let fcsDependencyManagerOptions =
+    let dummyPaketDependencyManagerOption =
+        match typeof<Marker>.Assembly.Location with
+        | "" -> ""
```

Comment:
Btw this might lead to an error in FCS as we will have the empty string in the argument array list. I'm not sure if FCS can deal with that.
Additionally, this should be `|null|"" ->` correct?

Reply:
I think what you wanted to do is use `[]` here and `[ sprintf "...` in the other branch

Reply:
Yes, you're right about the branch changes. I read the docs around Assembly.Location and it never returns null, only empty string in the worst case, so I felt comfortable using that directly here.

Reply:
Ah right I guess I have always checked for null or empty for nothing :D

Reply:
I don't blame you: the BCL has a habit of lying about things in type signatures 😢